rtgui/filmsimulation.cc

```diff
@@ -222,7 +219,13 @@ ClutComboBox::ClutComboBox(const Glib::ustring &path):
     set_model(m_model());
 
     if (cm->count > 0) {
-        pack_start(m_columns().label, false);
+		// Pack a CellRendererText in order to display long Clut file names properly
+		Gtk::CellRendererText *renderer;
+
+		renderer = new Gtk::CellRendererText;
```

Comment:
Wrap this in a [`Gtk::manage()`](https://developer-old.gnome.org/gtkmm/3.16/namespaceGtk.html#a753d0ddca83bd0d0decdfdcd1e93076a).